pages/common/greater-than.md

```diff
@@ -18,3 +18,7 @@
 - Redirect both `stdout` and `stderr` to `/dev/null` to keep the terminal output clean:
 
 `{{command}} &> /dev/null`
+
+- Ensure that a file is empty:
```

Comment:
```suggestion
- Clear the file contents:
```
It is more clear to me what it does that way

Reply:
I don't it's good enough. I considered that, but this command also allows creating a file. Maybe my description isn't good enough either.

Reply:
What about the updated one @managor?

Reply:
I was first talking about your suggestion and then I was talking about mine.

Reply:
I mean I updated by suggestion @managor

Reply:
Oh I see. Yes, It's good enough. I wanted a more compressed description but I guess it's impossible.